### PR TITLE
Add README On How Navigator Loads Its Modules

### PR DESCRIPTION
**Pull request by thomas-tosoni**
MEDIARITHMICS/mediarithmics-navigator/pull/61

### CODE REVIEW

File: NAVIGATOR_LOADING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+## How To Load The Admin Plugin
+
+Open the console in your browser and set the localStorage with the command below.
+
+    localStorage.plugins = "[{\"name\":\"admin\", \"moduleName\":\"admin.conf\", \"urn\":\"/admin\", \"setupFile\":\"main.js\"}]"
+
+Setup __app-configuration.js__ using the __app-configuration.js.template__ and put it in the __conf__ folder.  
+Start the admin-platform with `grunt serve` or use the `prod.sh` script to start it in production mode.
```
Comment (ndussart) [inline NAVIGATOR_LOADING.md]: I don't see prod.sh in the platform-admin repository

**Comment by ndussart** [comment]
Could you make a littre drawing that explains the loading as weel ?

File: NAVIGATOR_LOADING.md (hunk 1 above)
Comment (thomas-tosoni) [inline NAVIGATOR_LOADING.md]: :+1:

**Comment by ndussart** [comment]
OK for me : waiting for PR on platform-admin with the prod.sh

**Comment by thomas-tosoni** [comment]
There is no prod.sh, you can merge the pull request :grimacing:

**Pull request merged**